src/components/core/ApplicationHeader.tsx

```diff
@@ -159,6 +159,13 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 								/>
 							</div>
 						</div>
+
+						{/* Report Problem */}
+						<div className="relative cursor-pointer">
```

Comment:
relative, cursor-pointer is not needed, unless there's a specific reason you added it?